internal/generate/clusterserviceversion/clusterserviceversion_updaters.go

```diff
@@ -28,11 +29,19 @@ import (
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
+	apiextv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 
 	"github.com/operator-framework/operator-sdk/internal/generate/collector"
 	"github.com/operator-framework/operator-sdk/internal/util/k8sutil"
 )
 
+// serviceportPath is refers to the group of webhook service and
+// path names and port.
+type serviceportPath struct {
```

Comment:
For readability can you move this type to above the first function that uses it? It's irrelevant to the rest of this file.